kostyor/upgrades/drivers/openstack_ansible.py

```diff
@@ -0,0 +1,210 @@
+import os
+
+from ansible.cli.playbook import PlaybookCLI
+from ansible.executor.playbook_executor import PlaybookExecutor
+from ansible.inventory import Inventory
+from ansible.parsing.dataloader import DataLoader
+from ansible.vars import VariableManager
+
```

Comment:
The only concern I have here is based on the discussion on the ML recently:

http://lists.openstack.org/pipermail/openstack-dev/2016-November/106761.html
http://lists.openstack.org/pipermail/openstack-dev/2016-May/095489.html

We don't have a license in Kostyor currently but I don't want this decided for us by importing ansible. So, this driver needs to exist in a separate repo, and be loaded via Stevedore.